script8.js

```diff
@@ -0,0 +1,36 @@
+function change() {
+    window.location.href = 'https://www.google.com';
+}
+function goBack() {
+    window.history.back();
+}
+
+window.onload = function(){
+    document.getElementById("browser").innerHTML =  window.navigator.userAgent;
+    document.getElementById("version").innerHTML =  window.navigator.appVersion;
+    document.getElementById("height").innerHTML =  window.screen.height;
+    document.getElementById("width").innerHTML =  window.screen.width;
+    
+    document.getElementById('cookie').addEventListener('click',(e) =>{
+        let cookieName = "My Cookie";
+        let cookieValue = "Cookie Value";
+        document.cookie = cookieName + " = " + cookieValue;
+    });
+        
+    document.querySelector("#highlight").addEventListener('click', (e) => {
+        const paragraphs = document.querySelectorAll('p');
+
+        paragraphs.forEach((paragraphs) => {
+            paragraphs.classList.add('highlight');
+        });
+    });
```

Comment:
Its not a good idea to use the same variable name for the array and each iterable element in the array.
This will cause variable shadowing. Better would be 
```suggestion
    document.querySelector("#highlight").addEventListener('click', (e) => {
        const paragraphs = document.querySelectorAll('p');

        paragraphs.forEach((p) => {
            p.classList.add('highlight');
        });
    });
```